pkg/apis/aws/validation/infrastructure_test.go

```diff
@@ -165,7 +165,7 @@ var _ = Describe("InfrastructureConfig validation", func() {
 			It("should forbid empty zones", func() {
 				infrastructureConfig.Networks.Zones = nil
 
-				errorList := ValidateInfrastructureConfig(infrastructureConfig, &nodes, &pods, &services)
+				errorList := ValidateInfrastructureConfig(infrastructureConfig, []core.IPFamily{core.IPFamilyIPv4}, &nodes, &pods, &services)
```

Comment:
Just a small nit:
Since this array is used so many times in this file, I think it would be nice to create a var like:
`var familyIPv4 = []core.IPFamily{core.IPFamilyIPv4}`

Reply:
done